app/internal/window/window.go

```diff
@@ -14,13 +14,12 @@ import (
 	"gioui.org/unit"
 )
 
-type Options struct {
-	Width, Height       unit.Value
-	MinWidth, MinHeight unit.Value
-	MaxWidth, MaxHeight unit.Value
-	Title               string
+type Option struct {
```

Comment:
I suggest keeping the "Options" name; the type does all possible options, even though not all may be set.